modules/retail/pages/retail-formulas.adoc

```diff
@@ -51,33 +51,32 @@ Zone 2 is primarily needed for the correct determination of the FQDNs of the br
 * In the [guimenu]``File`` field, type the name of your configuration file.
 . In the [guimenu]``Start of Authority (SOA)`` section, use these parameters for Zone 1:
-* In the [guimenu]``Nameserver (Ns)`` field, use the FQDN of the  branch server (for example: [systemitem]``branchserver.branch1.example.org``).
+* In the [guimenu]``Nameserver (NS)`` field, use the FQDN of the  branch server (for example: [systemitem]``branchserver.branch1.example.org``).
 * In the [guimenu]``Contact`` field, use the email address for the domain administrator.
 * Keep all other fields as their default values.
-. In the [guimenu]``Records`` section, in subsection [guimenu]``A``, click btn:[Add Item] and use these parameters to set up an A record for Zone 1:
+. In the [guimenu]``Records`` section, in subsection [guimenu]``A``, use these parameters to set up an A record for Zone 1:
 * In the [guimenu]``Hostname`` field, use the hostname of the branch server (for example: [systemitem]``branchserver``).
 * In the [guimenu]``IP`` field, use the IP address of the branch server (for example, [systemitem]``192.168.1.1``).
-. In the [guimenu]``Records`` section, subsection [guimenu]``NS``, click btn:[Add Item] and use these parameters to set up an NS record for Zone 1:
+. In the [guimenu]``Records`` section, subsection [guimenu]``NS``, use these parameters to set up an NS record for Zone 1:
 * In the input box, use the hostname of the branch server (for example: [systemitem]``branchserver``).
-. In the [guimenu]``Records`` section, subsection [guimenu]``CNAME``, click on btn:[Add Item] and add the hostname of the branch server in each of these fields:
-* [systemitem]``tftp``
-* [systemitem]``ftp``
-* [systemitem]``dns``
-* [systemitem]``dhcp``
-* [systemitem]``salt``.  The [systemitem]`salt` CNAME should be the FQDN of the branch server's external interface for proxy functionality to work correctly.
+. In the [guimenu]``Records`` section, subsection [guimenu]``CNAME``, use these parameters to set up CNAME records for Zone 1:
+* In the [guimenu]``Key`` field, enter [systemitem]``tftp``, and in the [guimenu]``Value`` field, type the hostname of the branch server (for example: [systemitem]``branchserver``).
+* Click [guimenu]``Add Item``. In the [guimenu]``Key`` field, enter [systemitem]``ftp``, and in the [guimenu]``Value`` field, type the hostname of the branch server.
+* Click [guimenu]``Add Item``. In the [guimenu]``Key`` field, enter [systemitem]``dns``, and in the [guimenu]``Value`` field, type the hostname of the branch server.
+* Click [guimenu]``Add Item``. In the [guimenu]``Key`` field, enter [systemitem]``dhcp``, and in the [guimenu]``Value`` field, type the hostname of the branch server.
+* Click [guimenu]``Add Item``. In the [guimenu]``Key`` field, enter [systemitem]``salt``, and in the [guimenu]``Value`` field, type the FQDN of the branch server (for example: [systemitem]``branchserver.branch1.example.org``).
```

Comment:
You left out important bit about using external interface FQDN. Now there was a fix for suma to look for all FQDNs on the proxy, but I'll need to retest it. Give me a couple of days and I'll come back with the result.

Also note that bind automatically append domain name if the entered value does not end with a dot character. So the example above should be [systemitem]``branchserver.branch1.example.org.``